src/App.tsx

```diff
@@ -1,18 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
```

Comment:
You can write it in one line in this case
```suggestion
const items = getNumbers(1, 42).map(n => `Item ${n}`);
```